theurgy.lic

```diff
@@ -179,6 +191,14 @@ class TheurgyActions
     end
   end
 
+  def check_spells
+    relevant_spells = ['Bless', 'Glythtide\'s Gift']
+
+    relevant_spells.select do |spell|
```

Comment:
I think this will output 'spells' twice. You might consider using `reget` so you only need to output once. Not a big deal though.